docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -243,7 +243,6 @@ of selected rows from the dimension table.
 Limitations
 -----------
 
-* Dynamic filtering is currently implemented only for :doc:`/connector/hive` and :doc:`/connector/memory` connectors.
```

Comment:
Maybe add a note that support for dynamic filtering is documented in individual connector doc pages

Reply:
Not sure that should be in the limitation section though. Maybe in the intro paragraph of this file.

Reply:
I will review with @Jessie212 tomorrow and come up with a plan..

Reply:
> Not sure that should be in the limitation section though

i think for features like this we should be more "positive". I.e list things that work.

Instead of saying "hive connector doesn't support aggregation pushdown", "postgresql is not parallel", we could say "hive is parallel" and "postgresql supports aggregation pushdown". 

as we add more and more features, the feature/connector matrix is going to be more sparse, and so listings things that work is easier to maintain. and "easier" implies "more accurate when user reads this".

Reply:
Added paragraph in introduction with links.